src/constants/constants.js

```diff
@@ -2,5 +2,16 @@ export const FORECAST_PATHS = {
   baseUrl: '/api',
```

Comment:
I renamed file "constants.js" to "forecaApi.js".
I try to divide constants in separate files. Еhereby I think it will be more better to read and understand of which part do they belong to.

Reply:
Yup, that's totally fine to do so. But make sure you name those files accordingly.